File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/CloseHandlingPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.Form.Programming
{
    public partial class CloseHandlingPage : UserControl
    {
        public CloseHandlingPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Click event of the button.
        /// Opens the demonstrated form.
        /// </summary>
        private void mobjButton_Click(object sender, EventArgs e)
        {
            ClosedForm form = new ClosedForm();
            form.Closed += new EventHandler(form_Closed);
            form.ShowDialog();

        }

        /// <summary>
        /// Handles Closed event for the demonstrated form.
        /// </summary>
        private void form_Closed(object sender, EventArgs e)
        {
            Gizmox.WebGUI.Forms.Form form = sender as Gizmox.WebGUI.Forms.Form;
            MessageBox.Show(string.Format("Form {0} is closed!", form.Text));
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Programming/CloseHandlingPage.vb
Namespace CompanionKit.Controls.Form.Programming

	Public Class CloseHandlingPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles Click event of the button.
        ''' Opens the demonstrated form.
        ''' </summary>
        Private Sub mobjButton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjButton.Click
            Dim form As New ClosedForm
            AddHandler form.Closed, New EventHandler(AddressOf Me.form_Closed)
            form.ShowDialog()

        End Sub

        ''' <summary>
        ''' Handles Closed event for the demonstrated form.
        ''' </summary>
		Private Sub form_Closed(ByVal sender As Object, ByVal e As EventArgs)
            Dim form As Gizmox.WebGUI.Forms.Form = TryCast(sender, Gizmox.WebGUI.Forms.Form)
			MessageBox.Show(String.Format("Form {0} is closed!", form.Text))
		End Sub

	End Class

End Namespace